blocks/article-grid/article-grid.css

```diff
@@ -0,0 +1,126 @@
+/* use styles based on live milo libs */
+@import "https://main--milo--adobecom.hlx.live/libs/blocks/section-metadata/section-metadata.css";
+@import "https://main--milo--adobecom.hlx.live/libs/blocks/media/media.css";
```

Comment:
This is problematic, using `.live` URLS. The better approach would be to use libs utils and load them via JS. We don't want `.live` resources loading for production content.

I would almost opt for most of this CSS living in `https://github.com/adobecom/blog/blob/main/styles/styles.css` if this grid should work on all pages. Keeping only direct grid related css here. Media, banner, section-metadata, etc living in /styles/styles.css

Reply:
Thanks for the flag. Followed Momiko's advice on using `loadStyle` instead of `@import`.
Let me know if that needs to be updated further, thank you